### DISCUSSION TITLE
Install on Ubuntu ARM

### DISCUSSION THREAD

**Discussion by quarfie** [requarks/wiki#4804] [Help / Questions] [answered]
I'm trying to follow the steps at https://docs.requarks.io/install/ubuntu

But I'm doing it on an ARM system.

Under the Install Docket section I changed `arch=amd64` to `arch=arm64` and that seemed to work.

But I run into issues with

`docker create --name=wiki-update-companion -v /var/run/docker.sock:/var/run/docker.sock:ro --restart=unless-stopped -h wiki-update-companion --network=wikinet requarks/wiki-update-companion:latest`

which returned a warning:

`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

Later, I don't get any error when I run docker start wiki-update-companion however the server is not responding on port 80 even after 5 minute wait, so I'm thinking it has something to do with the architecture...?

**Comment by NGPixel** [comment, marked as answer]
You can't use the wiki-update-companion image on ARM.

You can check the logs of the wiki container using `docker logs wiki`

**Reply by quarfie** [reply]
Thanks. The logs showed permission related errors but I decided to nuke it and start over with an x86 instance so I can install everything.
